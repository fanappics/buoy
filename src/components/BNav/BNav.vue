<template>
  <nav>
  </nav>
</template>

<script>
import BButton from '../BButton'
export default {
  name: 'b-nav',
  components: [BButton]
}
</script>

<style lang="stylus" scoped>

</style>
