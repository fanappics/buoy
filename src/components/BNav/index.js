import BNav from './BNav.vue'

/* istanbul ignore next */
BNav.install = function install (Vue) {
  Vue.component(BNav.name, BNav)
}

export default BNav
