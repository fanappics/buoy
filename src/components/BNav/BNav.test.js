import { mount, shallow } from 'vue-test-utils'
import BNav from './BNav'

describe('BNav', () => {
  it('renders a button', () => {
    const wrapper = mount(BNav)

    expect(wrapper.html()).toMatchSnapshot()
  })
})
