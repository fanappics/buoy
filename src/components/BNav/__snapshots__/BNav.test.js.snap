// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BNav renders a button 1`] = `<nav></nav>`;
